pack/tsk/tskp.json

```diff
@@ -734,6 +1786,234 @@
+        "customization_options": [
+            {
+                "xp": 0,
+                "choice": "choose_skill",
+                "text_change": "replace",
+                "position": 0
+            },
+            {
+                "xp": 1,
+                "text_change": "append"
+            },
+            {
+                "xp": 1,
+                "text_change": "append"
+            },
+            {
+                "xp": 2,
+                "real_slot": "Body",
+                "text_change": "replace",
+                "position": 1
+            },
+            {
+                "xp": 2,
+                "choice": "choose_skill",
+                "text_change": "insert",
+                "position": 0
+            },
+            {
+                "xp": 3,
+                "choice": "choose_skill",
+                "text_change": "insert",
+                "position": 0
+            },
+            {
+                "xp": 3,
+                "text_change": "append"
+            },
+            {
+                "xp": 3,
+                "text_change": "replace",
+                "position": 3
+            }
+        ],
+        "customization_change": "Customizable. Chosen skill: _____\nYou may play Living Ink under the control of any investigator at your location.\nAt the start of your turn, you may choose not to remove 1 charge from Living Ink and ignore its ability for the remainder of the round.\nUses (5 charges).\nAdditional skill: _____\nAdditional skill: _____\n[reaction] After you reveal a chaos token with a symbol, exhaust Living Ink: Place 1 charge on it.\nYou get +2 to the chosen skill(s) and -1 to each other skill.",
+        "customization_text": "Chosen skill:\n□ <b>Shifting Ink.</b> You may play Living Ink under the control of another investigator at your location.\n□ <b>Subtle Depiction.</b> At the start of your turn, you may choose not to remove 1 charge from Living Ink and ignore its ability for the remainder of the round.\n□□ <b>Imbued Ink.</b> Living Ink enters play with 2 additional charges and takes up an arcane slot instead of a body slot.\n□□ <b>Eldritch Ink.</b> Choose another skill.\n□□□ <b>Eldritch Ink.</b> Choose another skill.\n□□□ <b>Macabre Depiction.</b> Living Ink gains: \"[reaction] After you reveal a chaos token with a symbol, exhaust Living Ink: Place 1 charge on it.\"\n□□□ <b>Vibrancy.</b> Living Ink grants an additional +1 to the chosen skills and -1 to each other skill.",
+        "deck_limit": 2,
+        "faction_code": "mystic",
+        "illustrator": "Kelley R. Harris",
+        "name": "Living Ink",
+        "pack_code": "tskp",
+        "position": 79,
+        "quantity": 2,
+        "slot": "Body",
+        "text": "Customizable. When you purchase Living Ink, choose a skill and circle it on its upgrade sheet.\nUses (3 charges).\nRemove 1 charge from Living Ink at the start of each of your turns. If Living Ink has no charges, discard it.\nYou get +1 to the chosen skill(s).",
```

Comment:
Note here I broke up the `Uses` onto two lines, so that its easier to 'customize' it when they make changes (one option gives it +2 charges, another changes the end of round effect).